corehq/apps/userreports/static/userreports/js/expression_evaluator.js

```diff
@@ -11,6 +11,7 @@ hqDefine('userreports/js/expression_evaluator', function () {
         self.expressionText = ko.observable(editor.getSession().getValue());
         self.error = ko.observable();
         self.result = ko.observable();
+        self.isEvaluating = ko.observable(false);
```

Comment:
nit: i am assuming the button is already disabled while this is happening.

Reply:
Not quite following - are you saying I don't need the `false` here? (which is true, although I like initializing observables for the sake of being explicit)

Reply:
no no. Not about that. Just saying that when this is set to true, the button is disabled to submit an evaluation.
I am not well versed with the UI so might not be using the right terms here.

Reply:
got it - at this point the button isn't disabled, it's only disabled after the user clicks it, while the ajax is happening

Reply:
Thanks